Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Account
+    {
+        private string[] _amounts;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove unassigned field '_amounts', or set its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)